chapters/part3/semantic-parsing-seq2seq.md

```diff
@@ -320,6 +320,29 @@ and it should give you the following numbers
 
 <exercise id="6" title="Decoding">
 
+Let us now look at what kinds of predictions the model makes. We'll use a `Predictor` to do so. Recall that we covered `Predictor`s in
+[an earlier chapter](/training-and-prediction#4). For our current purpose, we can directly use the
+[`Seq2SeqPredictor`](https://github.com/allenai/allennlp-models/blob/master/allennlp_models/generation/predictors/seq2seq.py) implemented in
+the `allennlp-models` repository, with the model archive we trained on our task.
+
+Let's try it now.
+
+<codeblock source="part3/semantic-parsing-seq2seq/predictor_source_easy" setup="part3/semantic-parsing-seq2seq/predictor_setup"></codeblock>
+
+Looks like the model's doing pretty well on these inputs. It has clearly learned the mapping from strings to symbols, and the order of operators and their arguments.
+
+Let's try something more interesting now.
+
+<codeblock source="part3/semantic-parsing-seq2seq/predictor_source_medium" setup="part3/semantic-parsing-seq2seq/predictor_setup"></codeblock>
```

Comment:
In these, you could keep the setup code how it is, if you really want, because we've already seen it.  Definitely show it the first time, though.